blog/2025-06-25-compact-deep-dive-part-2.mdx

```diff
@@ -12,13 +12,13 @@ date: 2025-06-25
 
 This blog post is part of the Compact Deep Dive series, which explores how Compact contracts work on the Midnight network.
 Each article focuses on a different technical topic and can be read on its own, but together they provide a fuller picture of how Compact functions in practice.
-[Compact Deep Dive - Part 1](https://docs.midnight.network/blog/compact) looked at the TypeScript API of the `compactc` compiler-generated contract implementation.
+[Compact Deep Dive - Part 1](https://docs.midnight.network/blog/compact) looked at the TypeScript API of the `compact` compiler-generated contract implementation.
 If you haven’t read it yet, we encourage you to start there.
 This article looks at how circuits and witnesses are actually implemented in JavaScript code generated by the Compact compiler.
 
 In part 1, we used the Bulletin Board tutorial contract as an example.
-We compiled it with the `compactc` compiler and started looking at the files generated by the compiler.
-We used version 0.24.0 of `compactc`.
+We compiled it with the `compact` compiler and started looking at the files generated by the compiler.
+We used version 0.24.0 of `compact`.
```

Comment:
We can say "version 0.24.0 of the Compact toolchain" or "version 0.24.0 of the Compact compiler" or "compiler version 0.24.0".